pkg/gadgets/socket-collector/tracer/tracer.go

```diff
@@ -27,8 +27,8 @@ import (
 	"github.com/vishvananda/netns"
 )
 
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang IterTCPv4 ./bpf/tcp4-collector.c -- -I../../.. -Werror -O2 -g -c -x c
-//go:generate go run github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang IterUDPv4 ./bpf/udp4-collector.c -- -I../../.. -Werror -O2 -g -c -x c
+//go:generate go run -mod vendor github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang IterTCPv4 ./bpf/tcp4-collector.c -- -I../../.. -Werror -O2 -g -c -x c
+//go:generate go run -mod vendor github.com/cilium/ebpf/cmd/bpf2go -target bpfel -cc clang IterUDPv4 ./bpf/udp4-collector.c -- -I../../.. -Werror -O2 -g -c -x c
```

Comment:
I am not sure about this change, I added it because it was listed on https://github.com/golang/go/issues/29516#issuecomment-454159127

Reply:
I saw you removed this. Did you find it was not necessary?

Reply:
It was breaking the build in docker (`make gadget-container`) and it works fine without.